src/benchmarklib/tpch/tpch_queries.cpp

```diff
@@ -912,11 +910,11 @@ size_t tpch_supported_queries[NUM_SUPPORTED_TPCH_QUERIES] = {
     // 3, /* Enable once we support Exists and Subselects in WHERE condition */
     4, 5, 6,
     // 7, /* Enable once CASE and arithmetic operations of Aggregations are supported */
-    8, 9
+    8, 9,
     // 10, /* Enable once we support Subselects in Having clause */
     // 11, /* Enable once we support IN */
-    // 12, /* Enable once we support nested expressions in Join Condition */
-    // 13, /* Enable once we support Case */
+    // 12,
+    13 /* Enable once we support Case */
```

Comment:
Yaay :)
The comment seems to be out-of-place though.

Reply:
It's as muddled as before. Please see for yourself - upon closer inspection, almost all of above justifications why the queries are disabled seem to be off-by-one or entirely out-of-sync with the query they are annotating.

Reply:
The array is zero indexed, maybe this is where this stems from?

Reply:
Looked at some, the explanations seemed to match. What baffles me is that you have enable TPCH-14 here (I think you wanted to enable TPCH-13), and systemTest (which runs the TPCH tests) didn't fail.

Reply:
Foo, tpch_test.cpp uses its own array of supported TPCH queries. Not your job I know, but can you fix this, so that in tpch_test.cpp

```// clang-format off
INSTANTIATE_TEST_CASE_P(TPCHTestInstances, TPCHTest, ::testing::Values(
  0,
  // 1, /* // Enable once we support Subselects in WHERE condition */
  2,
  // 3, /* Enable once we support Exists and Subselects in WHERE condition */
  4,
  5,
  6,
  // 7, /* Enable once CASE and arithmetic operations of Aggregations are supported */
  8,
  9
  // 10, /* Enable once we support Subselects in Having clause */
  // 11, /* Enable once we support IN */
  // 12, /* Enable once we support nested expressions in Join Condition */
  // 13, /* Enable once we support Case */
  // 14, /* Enable once we support Subselects in WHERE condition */
  // 15, /* Enable once we support Subselects in WHERE condition */
  // 16, /* Enable once we support Subselects in WHERE condition */
  // 17, /* Enable once we support Subselects in WHERE condition */
  // 18, /* Enable once we support OR in WHERE condition */
  // 19, /* Enable once we support Subselects in WHERE condition */
  // 20 /* Enable once we support Exists and Subselect in WHERE condition */
), );  // NOLINT
// clang-format on
```
becomes
```
// clang-format off
INSTANTIATE_TEST_CASE_P(TPCHTestInstances, TPCHTest, ::testing::ValuesIn(tpch_supported_queries), );  // NOLINT
// clang-format on
```

em-ba-rra-ssing

Reply:
OK fixed. I believe everybody had his share of face-palming by now 🤣

Reply:
@mrzzzrm - this isn’t the first time that people get confused about the zero-indexing here. Should we just introduce an empty select at position 0?

Reply:
#791